### PR TITLE
Fixed syntax bug

### LINKED ISSUE #1: PREPARE FRONTEND. Set up environment for frontend developers
Install CRA (create-react-app) for frontend .

Ensure app runs on npm start.

### PR DESCRIPTION
**Pull request by vector-ok**
## Description
Added Material Design Bootstrap component library 
Added Bootstrap 4 component library.
Designed Intro page.
Designed UI for Login.
Designed UI for Sign up.
Installed library for date picker.
Set up UI for Admin login
Fixed syntax bugs

Fixes #1 #2 #3 #7 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: betalife-app/package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
     "mdbreact": "^4.26.1",
     "popper.js": "^1.16.1",
     "react": "^16.13.1",
+    "react-date-picker": "^8.0.1",
+    "react-datepicker": "^2.16.0",
```
Comment (ekrresa) [inline betalife-app/package.json:15]: Why are there two date-picker libraries?

File: betalife-app/src/components/Intro.js [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+  }
+
+  doCollapse() {
+    this.setState({
+      collapse: !this.state.collapse
+    });
+  }
+
+  toggle = (nr) => () => {
+    if (nr === 2) {
```
Comment (ekrresa) [inline betalife-app/src/components/Intro.js:30]: What does nr represent please?

File: betalife-app/src/components/Intro.js [hunk 3]
```diff
@@ -0,0 +1,311 @@
+import React from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { MDBNavbar, MDBNavbarBrand, MDBNavbarNav, MDBNavbarToggler, MDBCollapse, MDBNavItem, MDBNavLink, MDBContainer, MDBMask, MDBView, MDBIcon, MDBBtn, MDBModal, MDBModalBody, MDBModalHeader, MDBInput, MDBFormInline} from "mdbreact";
+import { BrowserRouter as Router } from "react-router-dom";
```
Comment (ekrresa) [inline betalife-app/src/components/Intro.js:5]: I think its best if package imports are grouped away from file imports. Improves readability.

**Review by ekrresa — APPROVED** [review]
Great! Can you add screenshots of the pages built next time

File: betalife-app/package.json (hunk 1 above)
Comment (vector-ok) [reply betalife-app/package.json:15]: It's an initial design. When implementing logic, some things will definitely change to better fit the theme and flow of the whole work.

Of course there'll be a clean up of unused packages before deploy.

File: betalife-app/src/components/Intro.js (hunk 2 above)
Comment (vector-ok) [reply betalife-app/src/components/Intro.js:30]: Nice observation. I'll be adding comments going forward.

**Pull request merged**